pvactools/lib/output_parser.py

```diff
@@ -433,6 +512,23 @@ def prediction_methods(self):
 
         return sorted(list(methods))
 
+    def add_pretty_row(self, row, entries, prediction_method, pretty_method, suffix):
+        if prediction_method in entries:
+            # st = score type, s = score
+            for st, s in entries[prediction_method].items():
+                if st == 'mhcflurry_presentation_score':
+                    row['MHCflurryEL Presentation %s' % suffix] = s
+                elif st == 'mhcflurry_processing_score':
+                    row['MHCflurryEL Processing %s' % suffix] = s
+                elif st == 'mhcflurry_presentation_percentile':
+                    row['MHCflurryEL Presentation %s' % suffix] = s
+                else:
+                    row['%s %s' % (pretty_method, suffix)] = s
+                
+        else:
+            # when would this ever get called?
```

Comment:
To answer this question: not all alleles/lengths are supported by all prediction algorithms. So you could have individual prediction algorithm score/percentile fields where no results are available for an output row for some of the prediction algorithms chosen by the user.